style.css

```diff
@@ -1,10 +1,78 @@
 @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap");
+
+:root {
+  --primary-color: #134dc7;
+  --secondary-color: #172b4d;
+  --default-transition: all 0.3s ease-in-out;
+}
+
 * {
   margin: 0;
   padding: 0;
+  box-sizing: border-box;
 }
 
 body {
-  background-color: #ffffff;
+  background-color: #fff;
   font-family: "Poppins", sans-serif;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 20px;
+}
```

Comment:
This section is not matching the template design.

<img width="298" alt="Screen Shot 2022-08-18 at 8 40 18" src="https://user-images.githubusercontent.com/82426682/185409362-c8df0d93-6ca4-473c-887f-cd6d57fdc6ba.png">

<img width="194" alt="Screen Shot 2022-08-18 at 8 40 31" src="https://user-images.githubusercontent.com/82426682/185409421-9e391dd3-34ad-44b7-b76e-ff4cb40ad239.png">

Please fix it.

<img width="556" alt="Screen Shot 2022-08-18 at 8 40 51" src="https://user-images.githubusercontent.com/82426682/185409490-687518c4-b843-48bd-ae0f-44985d94b615.png">